src/plot.cpp

```diff
@@ -695,6 +693,10 @@ void draw_mesh_lines(Plot pl, ImageData& data)
     outer = 1;
     inner = 2;
     break;
+#ifdef __GNUC__
+  default:
+    __builtin_unreachable();
+#endif
```

Comment:
What do you think about making a macro that wraps both the `#ifdef` and the `__builtin_unreachable()`?  I think it'll look a little cleaner, and you can give it a more helpful name like `assert_unreachable()` maybe.

Reply:
Also, this would allow us to more easily modify the behavior of all the `__builtin_unreachable`s i.e. maybe we want the preprocessor to remove them if compiled without `-Doptimize=on` or possibly replace them with an assertion if `-Ddebug=on`

Reply:
Not a bad idea, I'm a fan of isolating the ifdefs.